engine/src/main/java/org/terasology/world/block/BlockRegions.java

```diff
@@ -53,7 +56,7 @@ public static BlockRegion createFromMinAndMax(int minX, int minY, int minZ, int
      * @return new block region
      */
     public static BlockRegion createFromCenterAndExtents(Vector3ic center, Vector3ic extents) {
-        return new BlockRegion().union(center).addExtents(extents);
+        return new BlockRegion(center).extend(extents);
```

Comment:
I'm not sure this is intuitive - while writing tests for this, I was wondering whether 
```java
BlockRegions.createFromCenterAndExtents(new Vector3i(), new Vector3i(0,0,0))
```
is valid or not, and what it should return. In the given implementation it _is valid_, as the extents are added to the center block, while the utility method might be interpreted in a way that "extents" refers to the final extents...